Documentation/cassandra.md

```diff
@@ -92,6 +92,84 @@ cluster = Cluster(['rook-cassandra-client.rook-cassandra.svc.cluster.local'])
 session = cluster.connect()
 ```
 
+## Configuring Cassandra
+
+In order to overwrite the `cassandra.yaml` configuration values you can specify a configmap using `RackSpec.configMapName`.
+You can add key-value pairs to the data section of the ConfigMap.
+
+### Configuration Options
+
+| Name     | Type    | Default |
+|----------|-------------|---------|
+| `cluster_name` | string | the name of your cluster as specified in the cluster crd |
```

Comment:
Please add backticks to the "type" column as well.
Additionally please format the markdown table as seen in https://github.com/rook/rook/blob/master/README.md#L80-L88 (e.g., Visual Code has a formatter for Markdown).